src/test/groovy/org/prebid/server/functional/model/request/activitie/ActivityRule.groovy

```diff
@@ -0,0 +1,47 @@
+package org.prebid.server.functional.model.request.activitie
+
+import com.fasterxml.jackson.annotation.JsonValue
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import lombok.Value
+
+import static org.prebid.server.functional.model.request.activitie.ActivityRule.Priory.DEFAULT
+
+@Value(staticConstructor = "of")
+@ToString(includeNames = true, ignoreNulls = true)
+@EqualsAndHashCode
+class ActivityRule {
+
+    Priory priority
+    Condition condition
+    boolean allow = true
+
```

Comment:
And corresponding to the doc we have [privacyRegs](https://github.com/prebid/prebid-server/issues/2591#:~:text=analytics%22%5D%0A%20%20%20%20%20%20%20%20%20%20%7D%2C%0A%20%20%20%20%20%20%20%20%20%20allow%3A%20false%0A%20%20%20%20%20%20%20%20%7D%2C%7B-,privacyRegs%3A%20%5B%22*%22,-%5D%20%20%20%20//%20future%20link%20to) field.